sycl/test-e2e/AddressSanitizer/use-after-free/quarantine-1.cpp

```diff
@@ -0,0 +1,54 @@
+// REQUIRES: linux, cpu
```

Comment:
could you please also update names of quarantine-1.cpp, quarantine-2.cpp and use-after-free-1.cpp
Replace 1/2 with something explaining the difference.

Reply:
done